src/ring_theory/power_series/basic.lean

```diff
@@ -1130,8 +1155,29 @@ lemma mul_inv_of_unit (φ : power_series R) (u : units R) (h : constant_coeff R
   φ * inv_of_unit φ u = 1 :=
 mv_power_series.mul_inv_of_unit φ u $ h
 
+open finset nat
+
+variables {A : Type*} [comm_ring A]
+
 end ring
 
+section comm_ring
+variables [comm_ring R]
+
+/-- The ring homomorphism taking a power series `f(X)` to `f(aX)`. -/
+noncomputable def eval_neg_hom : power_series R →+* power_series R :=
+eval_mul_hom (-1 : R)
+
+@[simp] lemma eval_neg_hom_X : @eval_neg_hom R _ X = -X :=
+begin
+  rw eval_neg_hom, ext, simp only [linear_map.map_neg], rw coeff_X, split_ifs,
+  { rw [h, eval_mul_hom], simp only [coeff_mk, mul_one, ring_hom.coe_mk, coeff_one_X, pow_one], },
+  { rw eval_mul_hom, simp, suffices f : (coeff R n) X = 0, {rw f, rw mul_zero,},
+    rw coeff_X, split_ifs, refl, },
+end
```

Comment:
I feel like it would be useful to also state this about `eval_mul_hom`:
```suggestion
@[simp] lemma eval_mul_hom_neg_one : eval_mul_hom (-1 : R) X = -X :=
begin
  ext, simp only [linear_map.map_neg], rw coeff_X, split_ifs,
  { rw [h, eval_mul_hom], simp only [coeff_mk, mul_one, ring_hom.coe_mk, coeff_one_X, pow_one], },
  { rw eval_mul_hom, simp, suffices f : (coeff R n) X = 0, {rw f, rw mul_zero,},
    rw coeff_X, split_ifs, refl, },
end

@[simp] lemma eval_neg_hom_X : @eval_neg_hom R _ X = -X :=
eval_mul_hom_neg_one 
```